composer.lock

```diff
@@ -4,30 +4,29 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file",
```

Comment:
Yes indeed. We'll need stricter version checks for some dependencies. I updated everything because there was some dependency resolve issue on my local and this seemed the easiest way. Tests are still passing and everything seems fine, so I would say this won't be an issue.